DesktopModules/Vanjaro/Core/Library/Data/Entities/WorkflowPermissionInfo.cs

```diff
@@ -13,4 +13,18 @@ public WorkflowPermissionInfo()
 
         }
     }
+
+    public class CustomPermissionInfo
```

Comment:
Will this class be reused for other entities in future? If so, it should be renamed to EntityPermissionInfo; otherwise, it should be named SectionPermissionInfo